docs/AA--Javascript-in-EE/value-objects/datetime.md

```diff
@@ -0,0 +1,718 @@
+
+If the provided value is not a valid offset, this will throw a `TypeError` error.
+
+#### `DateTime.validateIsDateTime( datetime )`
+
+Returns `boolean` to indicate whether the provided value is an instance of `DateTime`.
+
+#### `DateTime.assertIsDatetime( datetime )`
+
+If the provided value is not an instance of `DateTime`, this will throw an `InvalidDateTime` error.
+
+#### `DateTime.validateIsDate( date )`
+
+Returns `boolean` to indicate whether the provided value is an instance of the javascript `Date` object.
+
+#### `DateTime.assertIsDate( date )`
+
+If the provided value is not an instance of `Date`, this will throw a `TypeError` error.
+
+#### `DateTime.isValid( datetime )`
+
+Returns `boolean` to indicate whether the provided value is a _valid_ instance of `DateTime`.
+
+#### `DateTime.assertIsValid( datetime )`
+
+If the provided value is not an instance of `DateTime` or is not a _valid_ `DateTime`, this will throw an `InvalidDateTime` error.
+
+#### `DateTime.max( ...datetimes )`
+
+Given an indeterminate number of `DateTime` instances passed in as arguments, this will return a new instance of `DateTime` representing the latest moment of time in those arguments.
+
+**Example:**
+
+```js
+// maxDate will equal dtA (but is not the same instance as dtA)
+const dtA = new DateTime( '2018-12-22T10:15:00.200+00:00' );
+const dtB = new DateTime( '2017-12-25T10:15:00.200+00:00' );
+const dtC = new DateTime( '2018-11-25T10:15:00.200+00:00' );
+
+const maxDate = DateTime.max( dtA, dtB, dtC );
+```
+
+#### `DateTime.min( ...datetimes )`
+
+Given an indeterminate number of `DateTime` instances passed in as arguments, this will return a new instance of `DateTime` representing the earliest moment of time in those arguments.
+
+**Example:**
+```js
+// minDate will equal dtB (but is not the same instance as dtB)
+const dtA = new DateTime( '2018-12-22T10:15:00.200+00:00' );
+const dtB = new DateTime( '2017-12-25T10:15:00.200+00:00' );
+const dtC = new DateTime( '2018-11-25T10:15:00.200+00:00' );
+
+const minDate = DateTime.min( dtA, dtB, dtC );
+```
+
+#### `DateTime.fromISO( isoString, timezone, locale )`
+
+This returns an instance of `DateTime` from the given `ISOString`, timezone and locale.  It functions identically to the `DateTime.constructor`.
+
+#### `DateTime.fromISOWithOffset( isoString, offset, locale )`
+
+This returns an instance of `DateTime` from the given `ISOString`, offset, and locale.  It is similar in behaviour to `DateTime.fromISO` with the difference that an offset can be used instead of timezone string.
+
+#### `DateTime.fromJSDate( date, timezone, locale )`
+
+This receives a Javascript `Date` object along with timezone and locale strings to create (and return) an instance of `DateTime`.
+
+#### `DateTime.fromJSDateWithOffset( date, offset, locale )`
+
+This is identical in behaviour to `DateTime.fromJSDate` with the exception it receives an offset value instead of timezone string.
+
+#### `fromMillis( milliseconds )`
```

Comment:
how about `fromMilliseconds()`

Reply:
See my reply everywhere you've suggested this.